tests/treeTest.py

```diff
@@ -0,0 +1,18 @@
+from coalahtml.tree import Tree
+import unittest
+
+
+class coalaHTMLTest(unittest.TestCase):
+
+    def setUp(self):
+        self.FL = [
+                    "/home/tushar/coala-html/tests",
+                    "/home/tushar/coala-html/coalahtml",
+                    "/home/tushar/coala-html/setup.py",
+                    "/home/tushar/coala-html/tests/TestUtilities.py"
+                ]
```

Comment:
I think it would. Can you just confirm and let me know? I ran tests over
coala repo to see if it's working fine.  :)
On 22 May 2016 21:28, "Attila" <notifications@github.com> wrote:

> In tests/treeTest.py
> <https://github.com/coala-analyzer/coala-html/pull/46#discussion_r64151981>
> :
>
> > @@ -0,0 +1,18 @@
> > +from coalahtml.tree import Tree
> > +import unittest
> > +
> > +
> > +class coalaHTMLTest(unittest.TestCase):
> > +
> > +    def setUp(self):
> > +        self.FL = [
> > +                    "/home/tushar/coala-html/tests",
> > +                    "/home/tushar/coala-html/coalahtml",
> > +                    "/home/tushar/coala-html/setup.py",
> > +                    "/home/tushar/coala-html/tests/TestUtilities.py"
> > +                ]
>
> Will it work with multiple levels? Like several more in tests dir. I think
> it may not.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala-html/pull/46/files/3ccae34bd5a37e74f317493ac89a4dd2ce56133d..661e49c95350dc09646e3a4b15df6cf3fe2430de#r64151981>
>